plugins/Live/templates/_dataTableViz_visitorLog.twig

```diff
@@ -190,6 +190,7 @@ GPS (lat/long): {{ visitor.getColumn('latitude') }},{{ visitor.getColumn('longit
             {% endif %}
 
             <div class="col-md-{% if displayVisitorsInOwnColumn %}7{% else %}8{% endif %} column {% if visitor.getColumn('visitConverted') and not isWidget %}highlightField{% endif %}">
+                {% block beforeActionsInfo %}{% endblock %}
```

Comment:
I don't know if posting events in templates is a good idea. If you loose all advantages of twig templates.